ir_sequence_standard_default/models/ir_sequence.py

```diff
@@ -0,0 +1,11 @@
+# 2019  Vauxoo (<http://www.vauxoo.com/>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class IrSequence(models.Model):
+
+    _inherit = "ir.sequence"
+
+    implementation = fields.Selection(default='standard')
```

Comment:
Did you consider using an `ir.default` linked to a config setting?

For instance:
https://github.com/odoo/odoo/blob/d34d6867f2ff9abdf303f12b3fb9b7fc20e709ab/addons/sale/models/res_config_settings.py#L36

I'm not so sure if it would be convenient/usefull but, That way, the default value could be changed to its default behavior anytime in the future.

Reply:
Thank you, I took your idea and added it to the settings as a configurable value.

![Selección_782](https://user-images.githubusercontent.com/442938/69659949-ac778500-1055-11ea-8a1e-68490bcb8a4b.png)